src/include/txn.i

```diff
@@ -711,14 +711,14 @@ __wt_txn_upd_visible(WT_SESSION_IMPL *session, WT_UPDATE *upd)
 static inline int
 __wt_txn_read(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDATE *upd, WT_UPDATE **updp)
 {
-    static WT_UPDATE tombstone = {.txnid = WT_TXN_NONE, .type = WT_UPDATE_TOMBSTONE};
-    WT_DECL_RET;
-    WT_UPDATE *tmp_upd;
+    /* FIXME-PM1521: To be Removed by WT-5282 */
```

Comment:
Is it not better if you just remove the code in this ticket anyway?

Reply:
I'd like to. Vamsi has different opinion that.

Reply:
@bvpvamsikrishna please confirm.

Reply:
Yes, I asked Chenhao to comment out the code, for birthmarks instead of deleting it.

Reply:
Maybe I am thinking too much, @quchenhao  delete the code instead of commenting out.

Reply:
deleted